Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: build-tool
+build-tool:
+	go build main.go
```

Comment:
you don't need to add main.go at the end. With this specific target the created executable file will be `main`, instead of `json-schema-generator`

Reply:
Fixed.